src/containers/Layout/NotificationDrawerWrapper.tsx

```diff
@@ -1,42 +1,70 @@
 import {
+  Button,
   NotificationDrawer,
   NotificationDrawerBody,
   NotificationDrawerHeader,
   NotificationDrawerList,
   NotificationDrawerListItem,
   NotificationDrawerListItemBody,
-  NotificationDrawerListItemHeader
+  NotificationDrawerListItemHeader,
+  ButtonVariant
 } from '@patternfly/react-core';
-import React, { useContext } from 'react';
+import React, { useContext, useEffect, useState } from "react";
 import { AppContext } from '../../context/context';
+import { TimesIcon, TimesCircleIcon } from "@patternfly/react-icons";
+import { NotificationActionTypes } from "../../context/actions/types";
 
 interface NotificationDrawerWrapperProps {
   onClose: () => void;
 }
 
 const NotificationDrawerWrapper: React.FC<NotificationDrawerWrapperProps> = ({ onClose }) => {
-  const { state: { notifications } } = useContext(AppContext);
+  const [disabledClearAll, setDisabledClearAll] = useState(true);
```

Comment:
@colinywu 
I suggest renaming the variable to `isClearAllDisabled` in order to stick to the convention of using `is` and `has` as the prefixes for boolean flags. Please rename so out codebase is consistent.

```suggestion
  const [isClearAllDisabled, setIsClearAllDisabled] = useState(true);
```